src/app/extensions.cpp

```diff
@@ -51,12 +51,13 @@
 
 namespace app {
 
+const char* Extensions::kAsepriteDefaultThemeName = "aseprite-theme";
+
```

Comment:
No need of this change if the `ExtensionInfo` includes a bool to know if it overrides the default theme.